chef/site-cookbooks/wca/templates/worldcubeassociation.org.conf.erb

```diff
@@ -155,6 +155,45 @@ server {
     return 302 /media/;
   }
 
+  # Redirect e.php to results/events.php
+  location /results/e.php {
+    return 302 /results/events.php$is_args$args;
+  }
+  # Redirect events.php to /results/rankings
```

Comment:
nit: This is actually "Redirect /results/events.php to ..."

Reply:
Right, updated!